### ISSUE TITLE
Google Play Statistics 2019

### ISSUE THREAD

**Issue by westnordost** [westnordost/StreetComplete#1416]
I thought I share some statistics with you from Google Play like [I did in early 2018](https://github.com/westnordost/StreetComplete/issues/894)

Shown below is the metric **Installs on active devices**, which is the _number of Android devices that have been online at least once in the past 30 days that have your app installed._

How this metric is calculated changed in July 2018, reducing the number of users shown in the metric abruptly by about 20%. Compared to February 2018, StreetComplete has now about 600 more users, so about 12% more. So, this is why in this statistic, it appears as if there were now less users.

# By Country

The countries with the little flags have an own translation.

Country | Users | % of Total
-- | -- | --
Germany 🇩🇪| 1700| 31.1%
United States 🇺🇸| 613 | 11.2%
France 🇫🇷| 427 | 7.8%
United Kingdom 🇬🇧| 303 | 5.5%
Italy 🇮🇹| 227 | 4.2%
Australia | 134 | 2.5%
Poland 🇵🇱| 129 | 2.4%
Spain 🇪🇸| 126 | 2.3%
Switzerland | 115 | 2.1%
Austria | 114 | 2.1%
Netherlands 🇳🇱| 111 | 2.0%
Sweden 🇸🇻| 102 | 1.8%
Belgium | 90 | 1.6%
Russia 🇷🇺| 90 | 1.6%
Czechia 🇨🇿| 81 | 1.5%
Canada | 76 | 1.4%
Finland 🇫🇮| 56 | 1.0%
Denmark 🇩🇰| 51 | 0.9%
Brazil 🇧🇷| 45 | 0.8%
Ireland | 42 | 0.8%
Myanmar (Burma) | 42 | 0.8%
Argentina | 41 | 0.8%
Hungary 🇭🇺| 39 | 0.7%
Japan 🇯🇵| 39 | 0.7%
Portugal 🇵🇹| 39 | 0.7%
Norway 🇳🇴| 38 | 0.7%
Ukraine 🇺🇦| 36 | 0.7%
India | 33 | 0.6%
Bolivia | 28 | 0.5%
Iran 🇮🇷| 27 | 0.5%
Romania | 21 | 0.4%
New Zealand | 20 | 0.4%
Others | 430 | 7.9%
**Total** | **5465** | **100.0%**

What I find most noticeable is that there are now actually slightly *less* active users from Germany, both in percent and in absolute numbers. So, about +600 users in total but about -50 German users.

# By Android Version

Android Version | Users | % of Total
-- | -- | --
Android 9 | 2246 | 41.1%
Android 8.1 | 603 | 11.0%
Android 8.0 | 1087 | 20.0%
Android 7.1 | 331 | 6.1%
Android 7.0 | 538 | 9.8%
Android 6.0 | 421 | 7.7%
Android 5.1 | 92 | 1.7%
Android 5.0 | 61 | 1.1%
Android 4.4 | 77 | 1.4%
Android 4.3 | 4 | 0.07%
Android 4.2 | 5 | 0.09%

So < 4.4 is basically dead, however the big jump towards easier maintainability (and smaller download size) is 5.1. Looks like 4.4 will be around some more.

**Comment by rugk** [comment]
**Awesome!** And thanks for sharing.

Likely also a good idea to use the new "pin this issue" feature of GitHub for an issue like this one…
![image](https://user-images.githubusercontent.com/11966684/58829357-ef791280-8647-11e9-836d-1609740883c6.png)

Also great you've made the effort to find all these emojis for the countries. You missed, some, however, so I've used [my emoji picker add-on](https://addons.mozilla.org/de/firefox/addon/awesome-emoji-picker/?src=external-github) to add the missing ones. Here is a gist for you to copy: https://gist.github.com/rugk/f933e06d1718b883412558aab8219f6f

(Note: during copying your original emojis were deleted, so I only added those in those lines that were missing)
